graphics/vulkan-headers/Portfile

```diff
@@ -4,9 +4,9 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.0
 
-github.setup        KhronosGroup Vulkan-Headers sdk-1.1.101.0
+github.setup        KhronosGroup Vulkan-Headers v1.1.112
 name                vulkan-headers
-version             1.1.101.0
+version             v1.1.112
```

Comment:
We don't want the version to have the "v" in it. Remove the version line and change the `github.setup` line to be:

`github.setup        KhronosGroup Vulkan-Headers 1.1.112 v
`